tests/data/qa/multiqc_data/rnaaln/hisat2/multiqc.log

```diff
@@ -0,0 +1,46 @@
+[2024-06-14 17:14:50,959] multiqc.modules.hisat2.hisat2                      [INFO   ]  Found 1 reports
+[2024-06-14 17:14:50,959] multiqc.utils.util_functions                       [DEBUG  ]  Wrote data file multiqc_hisat2.txt
+[2024-06-14 17:14:50,964] multiqc.utils.software_versions                    [DEBUG  ]  Reading software versions from config.software_versions
+[2024-06-14 17:14:50,964] multiqc                                            [DEBUG  ]  Reordering sections: anchor 'icgc-argo-workflows-rnaaln-methods-description' not found.
+[2024-06-14 17:14:50,964] multiqc                                            [DEBUG  ]  Reordering sections: anchor 'software_versions' not found.
+[2024-06-14 17:14:50,964] multiqc                                            [DEBUG  ]  Reordering sections: anchor 'icgc-argo-workflows-rnaaln-summary' not found.
+[2024-06-14 17:14:50,964] multiqc                                            [DEBUG  ]  Reordering sections: anchor 'icgc-argo-workflows-rnaaln-methods-description' not found for module 'Picard'.
+[2024-06-14 17:14:50,964] multiqc                                            [DEBUG  ]  Reordering sections: anchor 'software_versions' not found for module 'Picard'.
+[2024-06-14 17:14:50,964] multiqc                                            [DEBUG  ]  Reordering sections: anchor 'icgc-argo-workflows-rnaaln-summary' not found for module 'Picard'.
+[2024-06-14 17:14:50,964] multiqc                                            [DEBUG  ]  Reordering sections: anchor 'icgc-argo-workflows-rnaaln-methods-description' not found for module 'HISAT2'.
+[2024-06-14 17:14:50,964] multiqc                                            [DEBUG  ]  Reordering sections: anchor 'software_versions' not found for module 'HISAT2'.
+[2024-06-14 17:14:50,964] multiqc                                            [DEBUG  ]  Reordering sections: anchor 'icgc-argo-workflows-rnaaln-summary' not found for module 'HISAT2'.
+[2024-06-14 17:14:50,978] multiqc.utils.util_functions                       [DEBUG  ]  Wrote data file multiqc_general_stats.txt
+[2024-06-14 17:14:50,978] multiqc                                            [DEBUG  ]  Compressing plot data
+[2024-06-14 17:14:50,994] multiqc.utils.util_functions                       [DEBUG  ]  Wrote data file multiqc_data.json
+[2024-06-14 17:14:50,995] multiqc                                            [INFO   ]  Report      : ICGC-ARGO-Post-RNA-Alignment-QC-Report_multiqc_report.html
+[2024-06-14 17:14:50,995] multiqc                                            [INFO   ]  Data        : ICGC-ARGO-Post-RNA-Alignment-QC-Report_multiqc_report_data
+[2024-06-14 17:14:50,995] multiqc                                            [DEBUG  ]  Moving data file from '/tmp/tmpgm0e_83_/multiqc_data' to '/Users/gfeng/Documents/Work/ICGC-ARGO-WORKFLOWS/rnaaln/update_prep_metrics/nf-core-rnaaln/work/db/878450a93307149b3ab3b8e29cae66/ICGC-ARGO-Post-RNA-Alignment-QC-Report_multiqc_report_data'
```

Comment:
Remove mentions of your own directory

Reply:
This is a auto-generated file from the previous step of prep metrics. In tests/data/qa/multiqc_data/dnaaln/hisat2/multiqc.log file it also has own directory. Maybe we can leave as they are?